manageability-docker/vision-docker/third-party-programs.txt

```diff
@@ -0,0 +1 @@
+../../third-party-programs.txt
```

Comment:
@gblewis1 - Do you have any recommendations on how we can include the third-party-programs.txt that we have at root with the docker files?  I don't know that this will do it.  We don't want to have duplicate documents.  Maybe just having it at the root level is enough.  What do you think?

Reply:
I talked to Gavin and we are just going to remove this since these aren't packaged.  Having the file at the root level is all we need.  Otherwise, it will cause mismatches.